x-pack/plugins/gis/public/shared/layers/sources/es_geohashgrid_source/es_geohashgrid_source.js

```diff
@@ -84,15 +83,21 @@ export class ESGeohashGridSource extends AbstractESSource {
     );
   }
 
-  renderDetails() {
-    return (
-      <ESSourceDetails
-        source={this}
-        geoField={this._descriptor.geoField}
-        geoFieldType="Point field"
-        sourceType={ESGeohashGridSource.typeDisplayName}
-      />
-    );
+  async getImmutableProperties() {
+    let indexPatternTitle = this._descriptor.indexPatternId;
+    try {
+      const indexPattern = await this._getIndexPattern();
+      indexPatternTitle = indexPattern.title;
+    } catch (error) {
+      // ignore error, title will just default to id
+    }
+
+    return [
```

Comment:
nice improvement to get consistent formatting.

Reply:
the only thing that might bite us if we want to do more fancy stuff depending on the source-type. E.g. ems-hotlinking is such a small edge-case. I feel like we take a step back with this:

![image](https://user-images.githubusercontent.com/1833023/51778971-8fd78d00-20d2-11e9-9792-f87d88160c05.png)

It used to say "Preview on Elastic Maps Landing Page", which is more descriptive